src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.RemoteServer.cs

```diff
@@ -95,9 +94,8 @@ public async Task SendAsync_SimpleGet_Success(Configuration.Http.RemoteServer re
             }
         }
 
-        [OuterLoop("Uses external servers")]
+        [OuterLoop("Uses external servers", TestPlatforms.Any & ~TestPlatforms.Browser)]
         [Theory, MemberData(nameof(RemoteServersMemberData))]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/53018", TestPlatforms.Browser)]
         public async Task SendAsync_MultipleRequestsReusingSameClient_Success(Configuration.Http.RemoteServer remoteServer)
```

Comment:
Do I understand this correctly? :
- Runs the tests on OuterLoop for non-browser cases
- Runs them with the regular builds for browser